javascript/javascript1/week2/student.js

```diff
@@ -0,0 +1,45 @@
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+    if (!studentName) {
+        console.log("please enter your name")
+        return
+    }
+    else{
+        if(class07Students.length < 6 || studentName==("Queen")){
+            if (class07Students.includes(studentName)){
+                console.log(`Student ${studentName} is already in the class`)
+                return
+            }
+            else{
+                class07Students.push(studentName)
+                return
+            }
+        }
+        else{
```

Comment:
this whole if chain could be simplified.  consider how 🤔

Reply:
this was for the first if

Reply:
I've made some changes